docs/src/ref/gfi.md

```diff
@@ -325,19 +325,36 @@ assess
 
 Most users can just use generative functions written in the [Built-in Modeling Language](@ref), and can skip this section.
 However, to develop new modeling DSLs, or optimized implementations of certain probabilistic modeling components, users can also implement custom types of generative functions.
+
+### Implementing a custom deterministic generative function
```

Comment:
Perhaps add a note here that conditioning on the value of a deterministic function in general is arbitrarily difficult (https://github.com/probcomp/Gen/issues/134#issuecomment-546660453) and therefore not an operation that `CustomDetGF` includes in its API?  (though could be implemented as an inference algorithm for a specific subclass where the inverse is known)

Reply:
Gen doesn't provide any means to constrain the return value of generative functions in general, even ones that are stochastic. So I think a note to that effect could be confusing here.

Reply:
Cool, sounds good.